python/lsst/sims/operations/Simulator.py

```diff
@@ -105,6 +108,8 @@ def __init__(self, lsstDB, obsProfile, sessionID, nRun, seeingEpoch, simStartDay
         self.telSeeing = telSeeing
         self.opticalDesSeeing = opticalDesSeeing
         self.cameraSeeing = cameraSeeing
+        self.scaleToNeff = scaleToNeff
+        self.atmNeffFactor = atmNeffFactor
```

Comment:
Just following the convention in the code of how to get parameters from one class to another. Doing it the right way would mean tearing a lot of things apart.